Sources/TuistKit/Services/Project/ProjectUpdateService.swift

```diff
@@ -55,7 +56,8 @@ struct ProjectUpdateService {
         _ = try await updateProjectService.updateProject(
             fullHandle: fullHandle,
             serverURL: serverURL,
-            defaultBranch: defaultBranch
+            defaultBranch: defaultBranch,
+            repositoryURL: repository
```

Comment:
The function's argument is called `repository`, but we pass it to the API as `repositoryURL`, so it feels a bit inconsistency. Should the API take a URL or a `tuist/tuist` slug? What's the most ergonomic option for developers? (thinking out loud)

Reply:
Changed it to `--repository-url`. I felt just `--repository` would be cleaner in the CLI context, but let's be more explicit and have the naming consistent with the API.